bundles/org.openhab.core/src/test/java/org/openhab/core/library/types/DateTimeGroupFunctionTest.java

```diff
@@ -36,36 +37,36 @@ public class DateTimeGroupFunctionTest {
 
     @Test
     public void testLatestFunction() {
-        ZonedDateTime expectedDateTime = ZonedDateTime.now();
+        Instant expectedDateTime = Instant.now();
```

Comment:
It was not necessary to change the tests, they would still pass. It was a consequence of https://github.com/openhab/openhab-core/pull/3583#discussion_r1865703710. If you prefer to keep `getZonedDateTime()` **and** prefer to use this in the tests, I can change it back. Let me know.

Reply:
Just to be 100% sure, I reverted and ran the tests, and as expected, they still passed. I would prefer to stick with this simplified version, because it's more to the point without unnecessary conversions:
- `ZonedDateTime.now()` creates a `ZonedDateTime` using `Instant.now()` with system default time-zone.
- `DateTimeType(ZonedDateTime)` strips the time-zone.
- `DateTimeType.getZonedDateTime()` return a `ZonedDateTime` using internal `Instant` with system default time-zone applied.

These conversions are already covered by `DateTimeTypeTest` methods, so I don't see the value of including them here also when testing the group function logic.